cmd/server.go

```diff
@@ -208,10 +224,13 @@ func main() {
 			if awsSession == nil {
 				if hc.Aws != nil && hc.Aws.Region != nil {
 					awsSession, err = session.NewSessionWithOptions(session.Options{
-						Config: aws.Config{Region: hc.Aws.Region},
+						Config:            aws.Config{Region: hc.Aws.Region},
+						SharedConfigState: session.SharedConfigEnable,
```

Comment:
is this sharedConfigState just making it easy to run locally?

Reply:
Yep, so that the Go AWS client will pick up environment variables or make calls out to the metadata service on an EC2 instance.